src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -869,14 +845,14 @@ TEST_CASE("Request rate - Shared memory infer input calls")
     ]
   }
       )"};
-  mdl->ReadDataFromStr(json_str, mmp->Inputs(), mmp->Outputs());
+  MockInputPipeline mip = TestLoadManagerBase::ProcessCustomJsonData(json_str);
 
   std::shared_ptr<ThreadStat> thread_stat{std::make_shared<ThreadStat>()};
   std::shared_ptr<RequestRateWorker::ThreadConfig> thread_config{
       std::make_shared<RequestRateWorker::ThreadConfig>(0, 1)};
 
-  trrm.parser_ = mmp;
-  trrm.data_loader_ = mdl;
+  trrm.parser_ = mip.mmp_;
```

Comment:
Can we name these better now?  mip is fine since it is defined immediately before, but mip.mmp means nothing to me.

Reply:
I second this. I think local variables whose fully-written form is too long are fine to shorten, but data members should be more verbose.

Reply:
done